server/backend/api/app/integration/forest_client_integration.py

```diff
@@ -8,7 +8,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class ForestClientService():
+class ForestClientIntegrationService():
```

Comment:
Renamed this service name to be the same name from the same service under: `server/admin_management`. So to keep them consistent. (we had two copies of the same service).